lib/pbench/test/functional/server/conftest.py

```diff
@@ -25,32 +27,46 @@ def server_client():
 
 
 @pytest.fixture(scope="module")
-def register_test_user(server_client: PbenchServerClient):
-    """Create a test user for functional tests."""
+def oidc_admin(server_client: PbenchServerClient):
+    """
+    Used by Pbench Server functional tests to get admin access
+    on OIDC server.
+    """
+    oidc_endpoints = server_client.endpoints["openid"]
+    oidc_server = OIDCAdmin(server_url=oidc_endpoints["server"])
+    return oidc_server
+
 
-    response = server_client.post(
-        API.REGISTER,
-        json={
-            "username": "tester",
-            "first_name": "Test",
-            "last_name": "User",
-            "password": "123456",
-            "email": "tester@gmail.com",
-        },
-        raise_error=False,
+@pytest.fixture(scope="module")
+def register_test_user(oidc_admin: OIDCAdmin):
+    """Create a test user for functional tests."""
+    response = oidc_admin.create_new_user(
+        username="tester",
+        email="tester@gmail.com",
+        password="123456",
+        first_name="Test",
+        last_name="User",
     )
 
     # To allow testing outside our transient CI containers, allow the tester
     # user to already exist.
     assert (
-        response.ok or response.status_code == HTTPStatus.FORBIDDEN
+        response.ok or response.status_code == HTTPStatus.CONFLICT
     ), f"Register failed with {response.json()}"
 
 
 @pytest.fixture
-def login_user(server_client: PbenchServerClient, register_test_user):
+def login_user(
+    server_client: PbenchServerClient, oidc_admin: OIDCAdmin, register_test_user
+):
     """Log in the test user and return the authentication token"""
-    server_client.login("tester", "123456")
-    assert server_client.auth_token
-    yield
-    server_client.logout()
+    oidc_endpoints = server_client.endpoints["openid"]
+    response = oidc_admin.user_login(
+        client_id=oidc_endpoints["client"], username="tester", password="123456"
```

Comment:
The username and password seem like "magic numbers" which should have a common definition and be accessed symbolically (e.g., here and at lines 44 and 46).